feat(frontend/about-us): style about-us page

winnteas opened the pull request:
added padding, nav bar and footer to aboutus

Commit:
added padding, nav bar and footer to aboutus

fafnirZ commented:
Hey winnie, would you be able to include a screenshot in your PR? as a rule of thumb paste screenshots for any UI change PRs just so reviewers don't need to pull your branch to check it out

winnteas commented:
![image](https://user-images.githubusercontent.com/89847626/191479906-b9cffa33-8291-48ff-848f-8c38a286fa09.png)

fafnirZ commented on next/src/pages/AboutUs.tsx:
```diff
@@ -1,64 +1,88 @@
+    <PageContainer>
+      {!navbarOpen && <Navbar open={navbarOpen} setNavbarOpen={handleToggle} />}
+
+      <div style={{ paddingLeft: "80px", paddingRight:"80px", paddingTop:"180px", paddingBottom: "180px" }}>
```
consider changing this to use styled components instead of inline styling,
furthermore, look into mobile compatibility and whether the padding will affect the text on a smaller screen

Commit:
removed in-line styling

Commit:
added mobile padding

winnteas commented:
- changed in-line styling to using styled components
- added mobile compatibility

Screenshots:
![Screen Shot 2022-10-17 at 9 05 43 pm](https://user-images.githubusercontent.com/89847626/196150507-c933e2d6-4ba9-45c9-99f5-b5bbc7428331.png)

![Screen Shot 2022-10-17 at 9 06 01 pm](https://user-images.githubusercontent.com/89847626/196150569-0c55bcb5-06f1-4085-bce2-6e54af64de79.png)
